ansible/files/.env.example

```diff
@@ -22,3 +22,15 @@ export SENDGRID_API_KEY=123
 
 # (optional) override devise hours after which link expires (defaults to 6)
 export HOURS_PASSWORD_RESET_LINK_EXPIRES_AFTER=8
+
```

Comment:
Just checking that you intended to be changing the `ansible` example file instead of the root `.env` one?

Reply:
@SMores - just wanted to check on this?